source/reference/api/progress.txt

```diff
@@ -33,8 +33,15 @@ The ``progress`` endpoint returns either an updated status or an error.
 Successful Response
 ~~~~~~~~~~~~~~~~~~~
 
+The response object contains 2 top-level fields, ``success`` and
+``progress``. 
+
+The ``success`` field contains the status of the ``progress`` command. 
+Value is ``true`` if the command succeeds and ``false`` if the command
+fails. 
+
```

Comment:
(nit) Since this is in the "successful response" section, we could maybe simplify by saying here that `success` will always be true here.